configs/base.js

```diff
@@ -0,0 +1,72 @@
+module.exports = {
+  extends: ["eslint:recommended", "plugin:prettier/recommended"],
+  plugins: ["@foxglove", "file-progress", "import"],
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: "module",
+  },
+  rules: {
+    // show progress while linting
+    "file-progress/activate": 1,
```

Comment:
This seems like a weird thing for a base config to enable. Maybe we leave this to the individual project?

Reply:
Do you think some foxglove projects will not want it? Trying to reduce the amount of boilerplate required for new repos.

Honest I'm baffled that this is not just a built in feature of eslint.

Reply:
Removed in https://github.com/foxglove/eslint-plugin/pull/9